index.html

```diff
@@ -7,10 +7,55 @@
     <title>Weather Report</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="styles/index.css" />
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:ital,wght@1,600&family=Raleway:wght@300&display=swap" rel="stylesheet">
 </head>
+    <header class="header">
+        <h1>Selene and Jen's Weather Site</h1>
+            Here's what it looks like in
+            <p id="displayCity"></p>
+    </header>
 <body>
+    <!-- TEMP SECTION -->
+    <section id="temp__section">
+        <h2>Temp</h2>
+        <p id="temp-content__section">
+            <span id="temp-counter__span"></span> temp!
+        </p>
+        <button id="temp_add_button">⬆</button>
+        <button id="temp_minus_button">⬇</button>
+    </section>
+
+    <!-- CITY INPUT SECTION -->
+    <section id="city__section">
+        <h2>City</h2>
+        <label for="cityInput">Enter city name:</label>
+        <input type="text" id="cityInput" name="cityName" value="">
+        <button onclick="saveCity()">Save</button>
+        <button id="resetButton" onclick="resetCity()">Reset</button>
+    </section>
+
+    <!-- GARDEN SECTION -->
+    <section id="garden__section">
+        <h2>Sky Garden</h2>                 
+        <p id="garden-content__section">
+            <!-- <span id="garden__span"></span> Garden! -->
+        </p>
+    </section>
+
+    <!-- SKY SECTION -->
+    <section id="sky__section">
+      <h2>Sky</h2>
+      <select id="skySelect">
+        <option>Sunny</option>
+        <option>Cloudy</option>
+        <option>Rainy</option>
+        <option>Snowy</option>
+      </select>
```

Comment:
[The `option` HTML tag should also have a `value` field](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) to make these options accessible to screen readers: 
```
<select id="sky-menu">
    <option value="sunny">Sunny</option>
    <option value="cloudy">Cloudy</option>
    <option value="rainy">Rainy</option>
    <option value="snowy">Snowy</option>
</select>
```